docs/LTS.md

```diff
@@ -43,9 +43,9 @@ A "month" is to be a period of 30 consecutive days.
 
 | OS      | Version                | Package Manager           | Node.js      |
 |---------|------------------------|---------------------------|--------------|
-| Linux   | Ubuntu 16.04           | npm                       | 10,12,14,16  |
-| Linux   | Ubuntu 16.04           | yarn,pnpm                 | 10,12        |
-| Windows | Windows Server 2016 R2 | npm                       | 10,12,14,16  |
-| MacOS   | macOS X Mojave 10.14   | npm                       | 10,12,14,16  |
+| Linux   | Ubuntu 20.04           | npm                       | 10,12,14,16  |
+| Linux   | Ubuntu 20.04           | yarn,pnpm                 | 10,12        |
+| Windows | Windows Server 2019    | npm                       | 10,12,14,16  |
+| MacOS   | macOS X Mojave 10.15   | npm                       | 10,12,14,16  |
```

Comment:
Keeping up with whatever `-latest` means here will be very difficult. I think we should either update our test matrix to be explicit, or change this text to indicate that we track the GH Actions latest list.

Reply:
Preference for either option @jsumners @mcollina ?

Reply:
i would go with github latest.